com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ApplicationStager.java

```diff
@@ -91,10 +91,12 @@ public StepPhase stageApp(DelegateExecution context, CloudControllerClient clien
         UUID packageGuid = uploadToken.getPackageGuid();
 
         stepLogger.info(Messages.STAGING_APP, app.getName());
-
-        context.setVariable(Constants.VAR_BUILD_GUID, client.createBuild(packageGuid)
-            .getMetadata()
-            .getGuid());
+        CloudBuild build = client.createBuild(packageGuid);
+        UUID buildGuid = build.getMetadata()
+                              .getGuid();
+        stepLogger.getProcessLogger()
```

Comment:
What about using this method: StepLogger::infoWithoutProgressMessage which is only logging into the step and the process log?